pkg/app/src/pages/account/[...param].tsx

```diff
@@ -5,16 +5,20 @@ import { useCurrentAccountState } from 'hooks/useCurrentAccountState'
 import AccountPageGeneralLayout from 'components/Account/generalLayout'
```

Comment:
rename file to [path].tsx.
You dont need a catch all route. 

https://nextjs.org/docs/routing/dynamic-routes